### ISSUE TITLE
Rust: Testing and debugging should be much earlier.

### ISSUE THREAD

**Issue by DerSaidin** [kamranahmedse/developer-roadmap#6079]
### Roadmap URL

https://roadmap.sh/rust

### Suggestions

Move "Testing" and "Debugging" from almost last, to just after modules and crates.

At least move them before the "Ecosystem and Libraries" divider.

There's no way someone should be doing web development etc. without tests.

**Comment by dansholds** [comment]
Makes sense. This will potentially be actioned when we migrate the Rust roadmap to the new editor.

**Comment by dansholds** [comment]
Update: Roadmap not migrated yet.

**Comment by nilbuild** [comment]
This has been fixed and will be published in the coming release. Thanks @DerSaidin!
